src/hooks/index.js

```diff
@@ -70,10 +82,16 @@ exports.hasRole = (context, userId, roleName) => {
  * @returns resolves if the current user has ANY of the given permissions
  */
 const hasPermission = (inputPermissions) => {
-	const permissionNames = (typeof inputPermissions === 'string') ? [inputPermissions] : inputPermissions;
+	const permissionNames =
```

Comment:
Codacy found an issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)